class.jetpack-jitm.php

```diff
@@ -43,7 +43,7 @@ function photon_msg() {
 					</svg>
 				</div>
 				<p>
-					<?php _e( 'Deliver super-fast images to your visitors that are automatically optimized for any device.', 'jetpack' ); ?>
+					<?php _e( 'Speed up your media library and save bandwidth costs by using a free content delivery network.', 'jetpack' ); ?>
```

Comment:
About "Speed up your media library": Photon isn't applied to the Media Library, though, it's only applied to images inserted into posts.